samples/artifacts/DataSummarization/NamedAggregation.siddhi

```diff
@@ -0,0 +1,67 @@
+@App:name("NamedAggregation")
+@App:description("Demonstrates how to create a named aggregation for better query performance.")
+
+/*
+Purpose:
+    This application demonstrates how to create the named aggregation for better query performance.In this example the named aggregation keeps the aggregation of events such as average of price and sum of price.For more information on windows please refer to "https://siddhi.io/siddhi/documentation/siddhi-5.0/#window" The 'group by' clause helps to perform aggregation on events grouped by a certain attributes.In this sample, the symbol information per items is aggregated.
+
+Prerequisites:
+    1) Save this sample
+
+Executing the Sample:
+    1) Start the Siddhi application by clicking on 'Run'
+    2) If the Siddhi application starts successfully, the following messages would be shown on the console
+        * NamedAggregation.siddhi - Started Successfully!
+        
+Testing the Sample:
+    1) Click on 'Event Simulator' (double arrows on left tab)
+    2) Click 'Single Simulation' (this will be already selected)
+    3) Select 'NamedAggregation' as Siddhi App Name
+    4) Select 'TradeStream' as StreamName
+    5) Provide attribute values
```

Comment:
Give a CSV which can be used to simulate events for at least a couple of hours

Reply:
0c298022e90d12e96fa96a6435c301871fc28edb This commit fixed the above correction